_examples/complete/config/override-fluent-bit.yaml

```diff
@@ -0,0 +1,90 @@
+## -- Node affinity for particular node in which labels key is "Infra-Services" and value is "true"
+affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: "eks.amazonaws.com/nodegroup"
+          operator: In
+          values:
+          - "critical"
+
+
+## -- Using limits and requests
+resources:
+  limits:
+    cpu: 150m
+    memory: 150Mi
+  requests:
+    cpu: 100m
+    memory: 90Mi
+
+podAnnotations:
+  co.elastic.logs/enabled: "true"
+
+
+# -- EKS Cluster Details
+# eks_configs:
```

Comment:
can we remove this if not needed

Reply:
yes, we can. I have added these details inside `addons/fluent-bit/main.tf` using `set_values`